tests/remote-cache/Earthfile

```diff
@@ -36,15 +36,14 @@ ARG --global EARTHLY_BUILD_ARGS="REGISTRY"
 ARG --global REGISTRY_CONFIG="
 [registry.\"$REGISTRY\"]
   http = true
-  insecure = true
 "
 
 COPY test.earth ./Earthfile
 
 all:
     BUILD +test1
-    #BUILD +test2
-    #BUILD +test3
+    BUILD +test2
```

Comment:
these were accidentally commented out in the initial VERSION 0.7 PR; I want to re-enable them to ensure the `insecure = true` removal doesn't break them.